engine/enginefuzz_test.go

```diff
@@ -444,16 +494,27 @@ func getSeries(ctx context.Context, q storage.Queryable) ([]labels.Labels, error
 
 func validateTestCases(t *testing.T, cases []*testCase) {
 	failures := 0
+	logQuery := func(c *testCase) {
+		for _, load := range c.loads {
+			t.Logf(load)
+		}
+		t.Logf("query: %s, start: %d, end: %d, interval: %v", c.query, c.start.UnixMilli(), c.end.UnixMilli(), c.interval)
+	}
 	for i, c := range cases {
 		if !cmp.Equal(c.oldRes, c.newRes, comparer) {
-			for _, load := range c.loads {
-				t.Logf(load)
-			}
-			t.Logf(c.query)
+			logQuery(c)
 
 			t.Logf("case %d error mismatch.\nnew result: %s\nold result: %s\n", i, c.newRes.String(), c.oldRes.String())
 			failures++
 		}
+		if c.oldStats.Samples.TotalSamples != c.newStats.Samples.TotalSamples {
```

Comment:
I think we use cmp.Equal or something like this normally here

Reply:
Dont see how this was resolved :see_no_evil:

Reply:
Oops. Sorry. I thought you meant the comparison of TotalSamplesPerStep below. Latest revision has addressed this.